src/coreclr/nativeaot/Runtime/threadstore.cpp

```diff
@@ -325,7 +303,7 @@ void ThreadStore::WaitForSuspendComplete()
 
 void ThreadStore::InitiateThreadAbort(Thread* targetThread, Object * threadAbortException, bool doRudeAbort)
 {
-    SuspendAllThreads(/* waitForGCEvent = */ false, /* fireDebugEvent = */ false);
+    SuspendAllThreads(/* waitForGCEvent = */ false);
     // TODO: consider enabling multiple thread aborts running in parallel on different threads
```

Comment:
Everything related to threadabort is also only for FuncEval.

Reply:
Looks like that would require undoing https://github.com/dotnet/corert/pull/4100/commits/a61fac9dce2b1873a61cbe70489d8c0ecdad3ad4. It's a bit convoluted, so maybe a separate pull request at another time.